source/help/policies/content-types.md

```diff
@@ -1,6 +1,7 @@
 # Policies for specific content types
 
-All submissions to arXiv must meet the standards for [moderation and scholarly interest](/help/moderation/index.html).
+The standards for [moderation and scholarly interest](/help/moderation/index.html) are the basis for
+acceptance (or not) of submissions. However, from experience with comparing different types of content to that standard we give the guidelines below on what types of content typically are and are not accepted.
```

Comment:
I'm not quite sure what the second sentence is trying to convey here. 

How about, "In general, the guidelines below represent what types of content typically are and are not accepted."

Reply:
sounds good